docs/src/components/Header.js

```diff
@@ -35,105 +35,134 @@ export default function Header({ colorScheme, onChangeColorScheme }: Props) {
-            >
-              <Text color="white">
-                <GestaltLink
-                  href="https://codesandbox.io/s/k5plvp9v8v"
-                  target="blank"
-                >
-                  <Box padding={2}>Playground</Box>
-                </GestaltLink>
-              </Text>
-            </Tooltip>
+        <Box display="none" mdDisplay="flex" alignItems="center">
+          <Tooltip
+            inline
+            text={isRTL ? 'Left-To-Right View' : 'Right-To-Left View'}
+          >
+            <IconButton
+              size="md"
+              accessibilityLabel="toggle page direction: Left-To-Right / Right-To-Left View"
+              iconColor="white"
+              dangerouslySetSvgPath={togglePageDirSvgPath}
+              onClick={toggleRTL}
+            />
+          </Tooltip>
+          <Tooltip
+            inline
+            text={
+              colorScheme === 'light' ? 'Dark-Mode View' : 'Light-Mode View'
+            }
+          >
+            <IconButton
+              size="md"
+              accessibilityLabel="toggle color scheme: light / dark mode views"
+              iconColor="white"
+              icon="workflow-status-in-progress"
+              onClick={() => onChangeColorScheme()}
+            />
+          </Tooltip>
+          <Tooltip
+            inline
+            text="Opens Codesandbox ready to start coding with Gestalt"
+          >
             <Text color="white">
               <GestaltLink
-                href="https://github.com/pinterest/gestalt"
+                href="https://codesandbox.io/s/k5plvp9v8v"
                 target="blank"
               >
-                <Box padding={2}>GitHub</Box>
+                <Box padding={2}>Playground</Box>
               </GestaltLink>
             </Text>
-          </Box>
-          <Box display="flex" mdDisplay="none" alignItems="center">
-            <IconButton
-              size="md"
-              accessibilityLabel={`${isSidebarOpen ? 'Hide' : 'Show'} Menu`}
-              iconColor="white"
-              icon="menu"
-              onClick={() => setIsSidebarOpen(!isSidebarOpen)}
-            />
-          </Box>
+          </Tooltip>
+          <Text color="white">
+            <GestaltLink
+              href="https://github.com/pinterest/gestalt"
+              target="blank"
+            >
+              <Box padding={2}>GitHub</Box>
+            </GestaltLink>
+          </Text>
+        </Box>
+        <Box display="flex" mdDisplay="none" alignItems="center">
+          <IconButton
+            size="md"
+            accessibilityLabel={`${isSidebarOpen ? 'Hide' : 'Show'} Menu`}
+            iconColor="white"
+            icon="menu"
+            onClick={() => setIsSidebarOpen(!isSidebarOpen)}
+          />
         </Box>
       </Box>
+    </Box>
+  );
+}
+
+export default function StickyHeader({
+  colorScheme,
+  onChangeColorScheme,
+}: Props) {
+  const isReducedHeight = () => window.innerHeight < 709;
```

Comment:
@AlbertCarreras 709 is definitely a magic number, I'm ok with this approach for now but it will break for other use cases.